networkx/drawing/nx_pylab.py

```diff
@@ -16,11 +16,13 @@
  - :func:`matplotlib.pyplot.scatter`
  - :obj:`matplotlib.patches.FancyArrowPatch`
 """
+import warnings
```

Comment:
Is this needed anymore?

Reply:
Looks like "warnings" is used once, but it is imported at the time it is needed.
```suggestion
```